atomic_defi_design/assets/languages/atomic_defi_ru.ts

```diff
@@ -5,16 +5,18 @@
     <name>QPlatformTheme</name>
     <message>
         <source>&amp;Yes</source>
-        <translation>Yes</translation>
+        <translation>&amp;Да</translation>
```

Comment:
should I write content here with or without `&` @ssakone  @Milerius ?

Reply:
without &

Reply:
![image](https://user-images.githubusercontent.com/24797699/126602880-fd813782-4c25-495b-b359-d7acad9bd1f1.png)

should I ignore the warning?